src/ast/query.rs

```diff
@@ -78,7 +95,7 @@ pub enum SetExpr {
     /// UNION/EXCEPT/INTERSECT of two queries
     SetOperation {
         op: SetOperator,
-        all: bool,
+        op_option: Option<SetOperatorOption>,
```

Comment:
This change make users change their code to use `Option<SetOperatorOption>`.

Reply:
@unvalley I'd name it set_quantifier, as it is the name used in ANSI [(1)](https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#set-quantifier).

[1] : https://jakewheat.github.io/sql-overview/sql-2016-foundation-grammar.html#set-quantifier

Reply:
I agree set_quantifier sounds good (along with `SetOperatorOption` --> `SetQuantifier`)

Reply:
I fiexed at [f6e8305](https://github.com/sqlparser-rs/sqlparser-rs/pull/689/commits/f6e83058b0b5a37de5d2c9b0ce473186a1294ed4)